includes/html/api_functions.inc.php

```diff
@@ -72,6 +72,19 @@ function api_not_found()
     return api_error(404, "This API route doesn't exist.");
 }
 
+function search_ports(\Illuminate\Http\Request $request)
+{
+    $search = $request->get('search');
```

Comment:
what happens if search is empty or an empty string?

Reply:
it searchs for %%

Reply:
doesn't that mean "no filter", so all ports are searched?
not sure if it's such good on thousands of ports which will be returned as result